public/stylesheet.css

```
@@ -264,16 +264,27 @@ a {
     margin: 0 0 1rem;
   }
 }
-
 .header__list-item {
-  font-family: var(--font-family-stylized);
-  font-size: 1rem;
-  font-weight: 600;
-  margin: 0 0.7rem;
+  height: 55px;
+  width: 55px;
+  background-color: var(--color-background);
+  fill: var(--color-button-active);
 }
-.header__list-item:hover {
-  color: var(--color-button-hover);
-  text-decoration: underline;
+
+.header__list-item:hover,
+.header__list-icon:focus {
+  cursor: pointer;
+  fill: var(--color-background);
+  background-color: var(--color-button-active);
+  border-radius: 50%;
+}
+
+.header__list-icon {
+  padding: 10px;
+  height: 55px;
+  width: 55px;
+  display: inline-block;
+  fill: inherit;
```

Why don't you just set the `fill` value directly here:

```suggestion
  fill: var(--color-button-active);
```

Rather then inheriting it from it's parent [where you specify the color without any effect(?) on line 271](https://github.com/simple-icons/simple-icons-website/blob/50f437a69847dd976ba02619b01f6902680b0cec/public/stylesheet.css#L271).

---

I don't know, less letters? it's the same effect. 
If you prefer it set directly, we can do it.

---

> I don't know, less letters?

By my count having:

```css
[...]
fill: var(--color-button-active);
[...]
fill: inherit;
[...]
```

Is more letter than 

```css
fill: var(--color-button-active);
```